src/Umbraco.Core/Services/MemberService.cs

```diff
@@ -590,10 +584,11 @@ public IEnumerable<XElement> GetPagedXmlEntries(long pageIndex, int pageSize, ou
             Mandate.ParameterCondition(pageIndex >= 0, "pageIndex");
             Mandate.ParameterCondition(pageSize > 0, "pageSize");
 
-            var uow = UowProvider.GetUnitOfWork();
-            using (var repository = RepositoryFactory.CreateMemberRepository(uow))
+            using (var uow = UowProvider.GetUnitOfWork())

```

Comment:
This method here is a Get method ... why do we need to get a non-readonly unit of work and why do we need to commit it after just reading from the repository?